GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -18,7 +18,7 @@ if [ -z $PACKAGEVERSION ]; then
   exit 1
 fi
 
-BUILDOUTPUTDIR=$4
+BUILDOUTPUTDIR=${4%/}
```

Comment:
Why do we need the `/}` for `BUILDOUTPUTDIR` but not for `SOURCEDIRECTORY`?

Reply:
`GVFS.Installer.Mac.csproj` uses build variable `OutputPath` to set `BUILDOUTPUTDIR`. `OutputPath` is originally set elsewhere, I think in `GVFS.cs.pros` line no. 7 with a trailing `/`.

In other places that use `OutputPath`, like in `GVFS.Installer.Windows.csproj` line no. 35, line no. 51, the same pattern exists.

Reply:
It seems error prone to have scripts expect directory path parameters in different formats.  I've filed #892 to clean this up.

cc: @jrbriggs for his thoughts on if this cleanup should happen in this PR or if it can wait.

Reply:
I think we can defer that cleanup.